src/test/ui/suggestions/issue-106443-sugg-clone-for-bound.rs

```diff
@@ -0,0 +1,20 @@
+#[derive(Clone)]
```

Comment:
This test can be marked as `// run-rustfix`, because after applying the suggestions the code will compile :)